src/mixins/meta.ts

```diff
@@ -127,3 +155,51 @@ export function metaTag(
 
   return metaTagsObject;
 }
+
+// ALernate locales generator
```

Comment:
Alternate*

Reply:
Damn typos! There was also an error in one of the TS type names... fixed it.